sae_bench/evals/mdl/main.py

```diff
@@ -235,7 +237,9 @@ def check_quantised_features_reach_mse_threshold(
     mse_losses: list[torch.Tensor] = []
 
     for i in range(1):
-        x_BSN = activations_store.get_buffer(config.sae_batch_size)[0]
+        x_BSN = activations_store.get_filtered_buffer(config.sae_batch_size)
```

Comment:
This will filter out special tokens if needed by the SAE, which is technically a change in behavior but likely this is desired behavior.